draft-ietf-cbor-time-tag.md

```diff
@@ -664,6 +677,7 @@ The initial contents are shown in {{tab-timescales}}.
 | Timescale | Value | Semantics            | Reference |
 | UTC       |     0 | UTC with POSIX Epoch | [RFCthis] |
 | TAI       |     1 | TAI with PTP Epoch   | [RFCthis] |
+| UT1       |     2 | UT1 with POSIX Epoch | [RFCthis] |
```

Comment:
Is the POSIX epoch defined for UT1?  What was the UT1 value at 1970-01-01T00:00Z (UTC)?  We probably need to say.